### ISSUE TITLE
Multiple classes?

### ISSUE THREAD

**Issue by mongoose54** [jocicmarko/ultrasound-nerve-segmentation#15]
@jocicmarko If my target images have multiple classes (i.e. 0, 1, 2, etc.) do I have to modify anything on the last last layer of the NN?

**Comment by jocicmarko** [comment]
If you have N classes, the last convolution layer should have N filters, and also a softmax activation function instead of a sigmoid.

**Comment by mrgloom** [comment]
Why we are not using softmax by default?
We are talking about multi class or multi target? is `dice_coef_loss` suitable for this without modification?

**Comment by mrgloom** [comment]
Update:
As I understand in terms of Keras we can use `softmax+categorical_crossentropy` for multi class classification task or `sigmoid+binary_crossentropy` for multi label assignment task.
